grpcclient/grpcclient.go

```diff
@@ -74,11 +87,10 @@ func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 }
 
 func (cfg *Config) Validate() error {
-	switch cfg.GRPCCompression {
-	case gzip.Name, snappy.Name, "":
-		// valid
-	default:
-		return errors.Errorf("unsupported compression type: %s", cfg.GRPCCompression)
+	supportedCompressors := []string{gzip.Name, snappy.Name}
```

Comment:
Empty string (no compression) is missing here.

Reply:
Thanks, fixed.